dev/eslint-plugin-guardian-frontend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-plugin-guardian-frontend",
   "main": "index.js",
-  "version": "2.0.0",
+  "version": "3.1.0",
   "scripts": {
     "test": "jest"
```

Comment:
hmm, not sure actually. we'd want to add eslint, requireindex etc too, and then we'd need to keep them in sync with main versions. then would we install these versions or main ones? a lot of questions...will leave for now i think